resources/assets/components/PostUploader/index.js

```diff
@@ -47,33 +47,32 @@ class ReportbackUploader extends React.Component {
   handleOnSubmitForm(event) {
     event.preventDefault();
 
-    const reportback = {
+    const post = {
       media: this.state.media,
-      caption: this.caption.value,
+      text: this.caption.value,
       campaignId: this.props.campaignId,
       campaignRunId: this.props.campaignRunId,
       northstarId: this.props.northstarId,
-      source: this.props.source,
       status: 'accepted',
     };
 
     const signup = this.props.signup;
 
     if (this.quantity.value) {
-      reportback.impact = this.quantity.value;
-      signup.quantity = reportback.impact;
+      post.impact = this.quantity.value;
```

Comment:
It might make sense to rename this to `quantity` since that's what it's called in the v3 API.